Tests/AzineTestCase.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Azine\MailgunWebhooksBundle\Tests;
```

Comment:
IMO we don't need to support the old version of PHPUnit. => please remove this class & replace all `AzineTestCase` with `\PHPUnit\Framework\TestCase`